### ISSUE TITLE
no longer works

### ISSUE THREAD

**Issue by nykez** [WilliamVenner/gmsv_workshop#6]
please update the bindings!!

**Comment by WilliamVenner** [comment]
Tested and appears to work on every platform

**Comment by WilliamVenner** [comment]
Goofball

**Comment by piqey** [comment]
`attempt to index global 'steamworks' (a nil value)`
I hate to break this to you, but it doesn't work for me either. Tested using a dedicated server on Windows, both 32- and 64-bit binaries/branches…
… Goofball
I might try to figure out how to compile your project, but I've got one of my own I'm working my head around at the moment.

**Comment by piqey** [comment]
Obviously, it could be my turn to be the goofball, but maybe there is something wrong(?)

**Comment by WilliamVenner** [comment]
You need to `require("workshop")`

**Comment by piqey** [comment]
So it _was_ my turn after all. Sorry to bother you. Definitely couldn't be doing what I'm doing without you.

**Comment by piqey** [comment]
So I did actually try that first before commenting here, but the project I'm working on is written in [Teal](https://github.com/teal-language/tl), and the way we've got it set up has the build scripts set to run every time we save a file, but `require` works differently in Teal; the compiler couldn't find "workshop.tl", which caused a compiler error. Consequently, the build script didn't complete, and the Lua didn't update. A goofy situation.
